lnrpc/rpc.proto

```diff
@@ -1643,6 +1643,18 @@ message OpenChannelRequest {
 
     /// Whether unconfirmed outputs should be used as inputs for the funding transaction.
     bool spend_unconfirmed = 12 [json_name = "spend_unconfirmed"];
+
+    /*
+    Upfront address is an optional address which specifies the address to which
+    funds should be paid out to upon cooperative close. This field may only be
+    set if the peer supports the option upfront feature bit (call getfeatures
+    with the node pubkey to check). The remote peer will only accept cooperative
+    closes to this address if it is set.
+
+    Note: If this value is set on channel creation, you will *not* be able to
+    cooperatively close out to a different address.
+    */
+    string upfront_address = 13[json_name = "upfront_address"];
```

Comment:
while we're here, should we also display the shutdown addresses in `listchannels`?